src/app/payment-options/new-payment-option/new-payment-option.component.ts

```diff
@@ -44,21 +49,26 @@ export class NewPaymentOptionComponent implements OnInit, AfterViewInit {
     }
 
     addNewBankAccountClicked() {
-        const iban: string = (<string>$('#iban-holder').val()).replace(/ /g, '');
-        let bankCode: string = (<string>$('#bankcode-holder').val());
-        if (bankCode.length === 0) {
-            bankCode = 'N/A';
-        }
-        const alias: string = (<string>$('#alias-holder').val());
-
         SpinnerUtil.showSpinner();
-        this.paymentService.createBankAccount(iban, bankCode, alias).subscribe(res => {
-            SpinnerUtil.hideSpinner();
-            this.router.navigate(['dash', 'payment_options']);
-        }, err => {
-            SpinnerUtil.hideSpinner();
-            displayBackendError(err);
-        });
+        const controls = this.bankAccountForm.controls;
+        const iban = controls['iban'].value;
+        const bankCode = controls['bankCode'].value;
+        const alias = controls['bankAccountAlias'].value;
```

Comment:
@mightymatth shouldn't const in JS be used when we have value that won't change?

If we're going to have an option to edit bank account information in future, this might be a problem.

Reply:
@Skafec this connects in no way to the fact whether bank info will change. The `const` attribute is only valid within function scope. Task for you - learn about function scopes and how `const`, `var` and other types refer within them

Reply:
@Skafec The const in this context is fully valid. The button has been clicked and no values will be modified until the end of the function scope. 
Task for you - learn about function scopes and how `var` and `const` relate to them.